docs/services/recommendationservice.md

```diff
@@ -61,7 +58,25 @@ block ends execution. This is done in the `get_product_list` function.
 
 ## Metrics
 
-TBD
+### Initialize meter provider
+
+The OpenTelemetry SDK is initialized in the `__main__` code block. This code
+will create a meter provider. Export
+endpoints, resource attributes, and service name are automatically set by the
+OpenTelemetry auto instrumentor based on environment variables.
+
+```python
+    meter = metrics.get_meter_provider().get_meter("recommendationservice")
+```
+
+### Auto-instrumented metrics
+
+The following metrics are available through auto-instrumentation, courtesy of the `opentelemetry-instrumentation-system-metrics`, which is installed as part of `opentelemetry-bootstrap` on building the recommendationservice Docker image:
+
+* runtime.cpython.cpu_time
+* runtime.cpython.memory
+* runtime.cpython.gc_count
+
```

Comment:
It would be great to have the custom metrics documented here as well.

Reply:
Added